src/elasticsearch_cleanup/main.py

```diff
@@ -120,16 +116,15 @@ def remove_duplicates_from_index(
         ignore_unavailable=True,
     )
 
-    documents, log_documents = tee(documents)
-    logger.info(f"Found '{len(list(log_documents))}' documents inside '{index}'")
-
     bulk_actions = get_bulk_actions(
         documents=documents,
         comparison_fields=comparison_fields,
     )
-    logger.info(f"Found '{len(bulk_actions)}' duplicated documents")
 
-    if len(bulk_actions) == 0:
+    bulk_actions, bulk_actions_test = tee(bulk_actions)
+    try:
+        next(bulk_actions_test)
+    except StopIteration:
         logger.info(f"No duplicates found in '{index}'")
         return
```

Comment:
let's just remove this part. we have logs for `Number of successful deletions` and `Number of failed deletions` so we'll know if not duplicates have been found